spec/assignable_values/active_record_spec.rb

```diff
@@ -1024,6 +1024,38 @@ def genres
           years.collect(&:humanized).should == ['The year a new hope was born', 'The year the Empire stroke back', 'The year the Jedi returned']
         end
 
+        context 'when inheriting from a model with defined assignable_values' do
+
+          let(:base_class) do
+            base_class = Song.disposable_copy do
+              assignable_values_for :year do
+                [1977, 1980, 1983]
+              end
+            end
+            base_class.singleton_class.send(:remove_method, :name) # otherwise name is overridden by disposable_copy, even after inheriting
+            base_class
```

Comment:
The whole purpose of `disposable_copy` is to create a subclass with the same `#name` as the parent.

If you don't want this, just use `Class.new(superclass) do ... end` here.